### PR TITLE
Add redirect of support/new to /support

### PR DESCRIPTION
**Pull request by baisa**
To stop bots from being able to raise spam tickets.

[Trello card] (https://trello.com/c/TrY9IEND/2300-dgu-form-spam)

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -39,7 +40,6 @@
 
   get "search/", to: "search#search"
 
-  resources :tickets, only: %i[new create]
   get "tickets/confirmation", to: "tickets#confirmation"
```
Comment (ChrisBAshton) [inline config/routes.rb:43]: I'd also remove this route - and add a redirect route for `https://www.data.gov.uk/tickets/new` -> `https://www.data.gov.uk/support` 🙏 thank you!

Comment (baisa) [reply config/routes.rb:45]: Does this look all right?

File: config/routes.rb [hunk 2]
```diff
@@ -39,7 +40,8 @@
 
   get "search/", to: "search#search"
 
-  resources :tickets, only: %i[new create]
+  get "tickets/new", to: redirect("https://www.data.gov.uk/support")
```
Comment (ChrisBAshton) [inline config/routes.rb:43]:
```suggestion
  get "tickets/new", to: redirect("/support")
```

I think that's correct 🙏

File: config/routes.rb (hunk 1 above)
Comment (ChrisBAshton) [reply config/routes.rb:45]: (Line 43 still needs deleting)

**Review by ChrisBAshton — APPROVED** [review]
LGTM 👍 please check the redirects work on integration before promoting to staging/production (as per https://docs.publishing.service.gov.uk/manual/data-gov-uk-deployment.html) - thanks!

**Pull request merged**